tests/unit/test_cli.py

```diff
@@ -120,28 +130,57 @@ def download(path: str) -> io.StringIO | io.BytesIO:
     return workspace_client
 
 
-def test_workflow(ws, caplog):
-    workflows(ws)
+@pytest.fixture
+def ws1() -> WorkspaceClient:
```

Comment:
this diff is too big. rename `ws1` back to `ws` and add new fixture only for `ws2`.